README.md

```diff
@@ -88,20 +88,25 @@ For an easier understanding of the used acronyms and special terms in our docume
-| [cwa-server]              | Backend implementation for the Apple/Google exposure notification API.       |
-| [cwa-ppa-server]          | Backend implementation for the privacy-preserving analytics server.          |
-| [cwa-verification-server] | Backend implementation of the verification process.                          |
-| [cwa-verification-portal] | The portal to interact with the verification server.                         |
-| [cwa-verification-iam]    | The identity and access management to interact with the verification server. |
-| [cwa-testresult-server]   | Receives the test results from connected laboratories.                       |
-| [cwa-log-upload]          | The log upload service is the counterpart of the log upload in the app.      |
+| Repository                 | Description                                                                       |
+| -------------------------- | --------------------------------------------------------------------------------- |
+| [cwa-documentation]        | Project overview, general documentation, and white papers.                        |
+| [cwa-app-ios]              | Native iOS app using the Apple/Google exposure notification API.                  |
+| [cwa-app-android]          | Native Android app using the Apple/Google exposure notification API.              |
+| [cwa-wishlist]             | Community feature requests.                                                       |
+| [cwa-website]              | The official website for the Corona-Warn-App.                                     |
+| [cwa-server]               | Backend implementation for the Apple/Google exposure notification API.            |
+| [cwa-ppa-server]           | Backend implementation for the privacy-preserving analytics server.               |
+| [cwa-verification-server]  | Backend implementation of the verification process.                               |
+| [cwa-verification-portal]  | The portal to interact with the verification server.                              |
+| [cwa-verification-iam]     | The identity and access management to interact with the verification server.      |
+| [cwa-testresult-server]    | Receives the test results from connected laboratories.                            |
+| [cwa-log-upload]           | Counterpart of the log upload in the app.                                         |
+| [cwa-quick-test-frontend]  | waiting for description                                                           |
+| [cwa-quick-test-backend]   | waiting for description                                                           |
+| [cwa-event-landingpage]    | Landing page for CWA which is opened if the user does not have the app installed. |
+| [cwa-event-qr-code]        | Utility to generate QR codes for Event Registration.                              |
+| [cwa-quicktest-onboarding] | Informations about onboarding procedure for partners of the Corona-Warn-App.      |
```

Comment:
Informations => Information

https://dictionary.cambridge.org/dictionary/english/information
"Information is an uncountable noun meaning ‘facts about someone or something’. It is often followed by about: …"

Maybe include also "rapid test" in the description, for instance:

"Documentation for integration of rapid test partners."

which is also a bit shorter.

Reply:
Done in https://github.com/corona-warn-app/cwa-documentation/pull/570/commits/d7ea0cc8a0f1fe8662fd9d1c95c2377374202fc7